src/coffee/directives/api/models/parent/markers-parent-model.coffee

```diff
@@ -13,8 +13,8 @@ angular.module("uiGmapgoogle-maps.directives.api.models.parent")
           constructor: (scope, element, attrs, map) ->
             super(scope, element, attrs, map)
             self = @
```

Comment:
markers to plurals
reBuildMarkers to rebuildAll
added createChildScopes
mapClusterToMarkerModels to mapClusterToPlurals
markerModelsUpdate to pluralsUpdate
create
pieceMealWindows to peaceMeal